examples/order_book/README.md

```
@@ -29,16 +29,23 @@ python 01_create_market.py
 python 02_place_orders.py
 ```
 
 **What it does:**
-- Places split limit orders (creates YES holdings, sells NO shares)
-- Places buy orders for YES at various price levels
-- Establishes initial market liquidity
+1. Places split limit orders to create YES holdings (needed for selling)
+2. Places LP-eligible paired orders: YES SELL + NO BUY at complementary prices
+
+**LP Rewards Eligibility:**
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)